mmocr/datasets/pipelines/textdet_targets/drrg_targets.py

```diff
@@ -0,0 +1,534 @@
+import cv2
+import numpy as np
+from lanms import merge_quadrangle_n9 as la_nms
+from numpy.linalg import norm
+
+import mmocr.utils.check_argument as check_argument
+from mmdet.core import BitmapMasks
+from mmdet.datasets.builder import PIPELINES
+from .textsnake_targets import TextSnakeTargets
+
+
+@PIPELINES.register_module()
+class DRRGTargets(TextSnakeTargets):
+    """Generate the ground truth targets of DRRG: Deep Relational Reasoning
+    Graph Network for Arbitrary Shape Text Detection.
+
+    [https://arxiv.org/abs/2003.07493]. This code was partially adapted from
+    https://github.com/GXYM/DRRG licensed under the MIT license.
+
+    Args:
+        orientation_thr (float): The threshold for distinguishing between
+            head edge and tail edge among the horizontal and vertical edges
+            of a quadrangle.
+        resample_step (float): The step size for resampling the text center
+            line.
+        min_comp_num (int): The minimum number of text components, which
```

Comment:
min_comp_num -> num_min_comp
max_comp_num -> num_max_comp

Reply:
Should other xxx_num be changed to num_xxx?